File: C/Program Mencetak Barisan Bilangan Ganjil Genap dalam Rentang Tertentu.c
#include <stdio.h>

int main() {
    int awal, akhir;

    printf("Program Mencetak Barisan Bilangan Ganjil Genap dalam Rentang Tertentu\n");
    printf("visit us hobingoding.com\n\n");
    // Input Nilai Awal
    printf("Input Nilai Awal : ");
    scanf("%d", &awal);
    // Input Nilai Akhir
    printf("Input Nilai Akhir : ");
    scanf("%d", &akhir);

    // Barisan Bilangan Genap
    printf("Bilangan Genap : ");
    for(int i = awal; i <= akhir; i++) {
        if(i % 2 == 0)
            printf("%d ", i);
    }

    // Barisan Bilangan Ganjil
    printf("\n");
    printf("Bilangan Ganjil : ");
    for(int i = awal; i <= akhir; i++) {
        if(i % 2 != 0)
            printf("%d ", i);
    }
    return 0;
}

File: Python/Program Mencetak Barisan Bilangan Ganjil Genap dalam Rentang Tertentu.py
print("Program Mencetak Barisan Bilangan Ganjil Genap dalam Rentang Tertentu")
print("visit us hobingoding.com")
print()
# Input Nilai Awal
awal = int(input("Input Nilai Awal : "))
# Input Nilai Akhir
akhir = int(input("Input Nilai Akhir : "))

# Barisan Bilangan Genap
print("Bilangan Genap :", end=" ")
for i in range(awal  + 1, akhir + 1, 2):
    print(i, end=" ")

# Barisan Bilangan Ganjil
print("\nBilangan Ganjil :", end=" ")
for i in range(awal, akhir, 2):
    print(i, end=" ")